qemu/tests/cfg/vdpa_pxe_boot.cfg

```diff
@@ -0,0 +1,16 @@
+- vdpa_pxe_boot:
+    type = vdpa_pxe_boot
+    images = pxe
+    image_boot_pxe = no
+    image_name_pxe = images/pxe-test
+    image_size_pxe = 1G
+    force_create_image_pxe = yes
+    remove_image_pxe = yes
+    pxe_timeout = 60
+    kill_vm = yes
+    bootindex_nic1 = 1
+    boot_menu = on
+    machine_type_extra_params = "graphics=off"
```

Comment:
Not sure if `display = nographic` and `vga = none` would be better.

Reply:
I will try to test them and choose a better one to update.